src/renderer/apis/commands.ts

```diff
@@ -14,15 +14,87 @@ import { ApplicationCommandOptionType } from "../../types";
 import { constants, i18n, messages, users } from "../modules/common";
 import type { Store } from "../modules/common/flux";
 import { Logger } from "../modules/logger";
-import { getByStoreName } from "../modules/webpack";
+import { filters, getByStoreName, waitForModule } from "../modules/webpack";
 
 const logger = Logger.api("Commands");
 
+let RepluggedUser: User;
+
 interface CommandsAndSection {
   section: RepluggedCommandSection;
   commands: Map<string, AnyRepluggedCommand>;
 }
 
+declare class UserClass implements User {
```

Comment:
This is not needed. Just define the type of the module that you are waiting for below as `typeof User`.